cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/tasks/CassandraTaskExecutor.java

```diff
@@ -254,6 +256,8 @@ public CassandraTaskExecutor update(final ExecutorConfig config) {
                         config.getArguments(),
                         config.getURIs(),
                         ImmutableMap.<String, String>builder()
+                                .put("LD_LIBRARY_PATH", "$MESOS_SANDBOX/libmesos-bundle/lib")
```

Comment:
Looks like they can just be removed as they're already being `export`ed via scheduler.yml below.